packages/ecc-utils-design/src/components/form/form.ts

```diff
@@ -30,14 +30,16 @@ export interface Field {
     | "array"
     | "switch"
     | "file"
-    | "group";
+    | "group"
+    | "select";
   fieldOptions?: {
     required?: boolean;
     default?: string | boolean;
     multiple?: boolean;
     accept?: string;
     returnIfEmpty?: string;
     tooltip?: string;
+    options?: Array<{ label: string; value: string }>; 
```

Comment:
i dont think the `fieldOptions` object is the right place for this, as this object is for general purpose options that apply to pretty much all field types. Perhaps you can create a new object called `selectOptions` for these options.